nixos/modules/programs/screen.nix

```
@@ -22,20 +23,23 @@ in
     };
   };
 
-  config = {
-    # TODO: Added in 24.05, remove before 24.11
-    assertions = [
-      {
-        assertion = cfg.screenrc != null -> cfg.enable;
-        message = "`programs.screen.screenrc` has been configured, but `programs.screen.enable` is not true";
-      }
-    ];
-  } // lib.mkIf cfg.enable {
-    environment.etc.screenrc = {
-      enable = cfg.screenrc != null;
-      text = cfg.screenrc;
-    };
-    environment.systemPackages = [ cfg.package ];
-    security.pam.services.screen = {};
-  };
+  config = lib.mkMerge [
+    (lib.mkIf true {
+      # TODO: Added in 24.05, remove before 24.11
+      assertions = [
+        {
+          assertion = cfg.screenrc != null -> cfg.enable;
+          message = "`programs.screen.screenrc` has been configured, but `programs.screen.enable` is not true";
+        }
+      ];
+    })
```

```suggestion
    {
      # TODO: Added in 24.05, remove before 24.11
      assertions = [
        {
          assertion = cfg.screenrc != null -> cfg.enable;
          message = "`programs.screen.screenrc` has been configured, but `programs.screen.enable` is not true";
        }
      ];
    }
```
Having a condition around a hardcoded true or false, can always be reduced